code/android-core-library/src/test/java/com/adobe/marketing/mobile/launch/rulesengine/LaunchTokenFinderTest.kt

```diff
@@ -18,8 +17,9 @@ import com.adobe.marketing.mobile.MobileCore
 import com.adobe.marketing.mobile.SharedStateResult
 import com.adobe.marketing.mobile.SharedStateStatus
 import com.adobe.marketing.mobile.internal.eventhub.EventHub
-import com.adobe.marketing.mobile.internal.utility.TimeUtil
+import com.adobe.marketing.mobile.utils.TimeUtil
 import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNotNull
 import org.junit.Assert.assertNull
 import org.junit.Assert.assertTrue
 import org.junit.Before
```

Comment:
You fixed this class to not extend from `BaseTest` in your previous PR right? Did it get overwritten while resolving conflicts? Can we remove it?

Reply:
Yes, the git merge added it back, let me remove it now.